tests/pa/test_provision.py

```diff
@@ -146,8 +146,12 @@ def test_provision_telemetry_success(self, mock_util, distro_name, distro_versio
 
         ph.run()
 
-        call1 = call("Provisioning succeeded", duration=ANY, is_success=True)
-        ph.report_event.assert_has_calls([call1])
+        self.assertEqual(1, ph.report_event.call_count)
+        args = ph.report_event.call_args_list[0]
```

Comment:
Since you've asserted that report_event() was called exactly once, you can simplify this to be `args = ph.report_event.call_args` which is the call object representing the last (and, as verified, only) call to the mocked method.